// Copyright 2018 The Chromium OS Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto3";

package modemfwd;

enum JournalEntryType {
  UNKNOWN = 0;
  MAIN = 1;
  CARRIER = 2;
  OEM = 3;
}

message JournalEntry {
  string device_id = 1;
  string carrier_id = 2;
  repeated JournalEntryType type = 3;
};
